### DISCUSSION TITLE
Radio group with useController

### DISCUSSION THREAD

**Discussion by andrconstruction** [react-hook-form/react-hook-form#4776] [General] [answered]
Hi/ Is possible create a readio group with the useController hook? All my tryes tryes is failed Have you example for it with default values and so on

**Comment by deniercounter** [comment, marked as answer]
Please don’t expect an answer to such a general question. I would suggest you to provide a CodeSandBox that shows your implementation. This enables the community to have a quick look.

**Reply by andrconstruction** [reply]
Unfortunately I couldn't because all my tryes failed. Solved by another way , via common Controller

**Reply by deniercounter** [reply]
Perfect .. so you found your workaround. Happy coding.
